.github/workflows/pages.yml

```
@@ -0,0 +1,59 @@
+name: Reusable tailwind-nextjs-starter-blog workflow
+
+on:
+  workflow_call:
+    inputs:
+      base_path:
+        description: 'Base path URL for the blog'
+        required: false
+        default: ''
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: abernier/tailwind-nextjs-starter-blog
+          ref: pages
```

I'm using my fork/branch here for testing purposes, but will obviously become:

```suggestion
          repository: timlrx/tailwind-nextjs-starter-blog
          ref: main
```